services/provider/main.go

```diff
@@ -13,6 +17,58 @@ var (
 
 func main() {
 	flag.Parse()
-	var opts []grpc.ServerOption
-	server.Start(*port, opts)
+
+	klog.Info("Starting Provider API server")
+
+	namespace, err := util.GetWatchNamespace()
+	if err != nil {
+		klog.Errorf("failed to get provider cluster namespace. Error %v", err)
```

Comment:
```suggestion
		klog.Errorf("failed to get provider cluster namespace. %v", err)
```

The logger is already configured to log Error level so we don't need to explicitly add this.

Reply:
done.